### ISSUE TITLE
[HW2] user deleting and editing 

### ISSUE THREAD

**Issue by yuyupopo** [swsnu/swppfall2017#47]
Can a user delete or edit articles that is not created by him/her?
I think the specifications say that comments are like that.
Should it have to be the same for articles?

Thank you.

**Comment by wonook** [comment]
Yes, only the author of the article/comment should be able to see the edit/delete buttons.

**Comment by Kluckmuck** [comment]
So the author of the article can edit/delete alla comments, while the author of the comment can only edit/delete her own?

**Comment by wonook** [comment]
No, it means that the author of the article can edit/delete their articles and the author of comments can edit/delete their comments.
